test/e2e/apps/statefulset.go

```diff
@@ -242,7 +248,8 @@ var _ = SIGDescribe("StatefulSet", func() {
 			sst.WaitForRunningAndReady(*ss.Spec.Replicas, ss)
 		})
 
-		It("should perform rolling updates and roll backs of template modifications", func() {
+		// This is ok for Conformance because it doesn't use any PVCs.
+		framework.ConformanceIt("should perform rolling updates and roll backs of template modifications", func() {
```

Comment:
You need to add the appropriate conformance comment. See the README in the conformance dir. The same goes for all selected conformance tests.